ci/cron/perf/compare.sh

```diff
@@ -3,7 +3,7 @@
 
 #!/usr/bin/env bash
 
-set -euo pipefail
+set -euox pipefail
```

Comment:
left in deliberately, there is too much git commit changing in there that is confusing, that I think the extra verbosity is well worth it.